### DISCUSSION TITLE
Docker

### DISCUSSION THREAD

**Discussion by Kreger80** [merbanan/rtl_433#2620] [General]
I am working on this code and get it to run correctly keep getting an error.  Any help would be greatly appreciated 
![image](https://github.com/merbanan/rtl_433/assets/144958537/dfbf5c41-f688-436d-98c6-998cc8b3798c)
![image](https://github.com/merbanan/rtl_433/assets/144958537/d92997e5-5860-46bf-87a3-62521ff6cb1d)

**Comment by zuckschwerdt** [comment]
Is that a semicolon `;` instead of a colon `:` on the docker-compose.yml's `version` line?

And more importantly: what's this got to do with the rtl_433 project?

**Reply by Kreger80** [reply]
Yes I fixed that and it still didn’t work I do apologize I thought this was a general help page not specific
